### ISSUE TITLE
Cannot find empty_view

### ISSUE THREAD

**Issue by otakutyrant** [huhu/cpp-search-extension#20]
When I input `empty_view`, the extenstion does not show the complement as `std::ranges::empty_view` automatically.

**Comment by Folyd** [comment]
Probably it's a C++ 20 API? This extension was published in 2020, we didn't support C++ 20 yet.

**Comment by otakutyrant** [comment]
Yep. It is 2022 now, time for the new C+20 support! Looking forward to the new support. 🥹
